### ISSUE TITLE
VoI admission test for MACIDs

### ISSUE THREAD

**Issue by sbenthall** [causalincentives/pycid#59]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
The test for VoI admission currently does not work for multi-agent games.

https://github.com/causalincentives/pycid/blob/5f78703de2fe1ac4ba6864e68f5af02ad923f38b/pycid/analyze/value_of_information.py#L17-L20

It would be nice if this test also worked for models with more than one agent.

**Comment by RyanCarey** [comment]
Yes, this is a very desirable feature. However, I don't know any good algorithm to compute that quantity in a multi-agent setting. Unless there's a paper I don't know about, this will have to wait for us to do more theoretical research before we can build it.

**Comment by sbenthall** [comment]
Is the admissibility criterion the same?

**Comment by RyanCarey** [comment]
I think that might depend 1) how we define VoI, 2) whether you include the ways that your policy might affect others policy, and 3) what rationality assumptions we place on the other agents. The answer will be yes for some answers to (1-3), but I'm not sure how we should answer such questions. Sorry if that's not a clear answer.

**Comment by sbenthall** [comment]
Thank you -- I see now that the question is not entirely well-formed.
